api/lib/source/cosmosV0-source.js

```diff
@@ -356,10 +356,23 @@ class CosmosV0API extends RESTDataSource {
       this.getDelegationsForDelegatorAddress(address)
     ])
     const balances = balancesResponse || []
-    const coins = balances.map((coin) => {
+    let coins = balances.map((coin) => {
```

Comment:
Why let?

Reply:
because I overwrite it some lines later with `coins.push(...rewardsCoins`)`

Reply:
.push doesnt overwrite the object

Reply:
```suggestion
    const coins = balances.map((coin) => {
```

Reply:
ah ok, `push` doesn't overwrite the variable. I see